Doc/library/asyncio-policy.rst

```diff
@@ -7,22 +7,29 @@
 Policies
 ========
 
-An event loop policy is a global per-process object that controls
-the management of the event loop. Each event loop has a default
-policy, which can be changed and customized using the policy API.
-
-A policy defines the notion of *context* and manages a
-separate event loop per context. The default policy
-defines *context* to be the current thread.
-
-By using a custom event loop policy, the behavior of
-:func:`get_event_loop`, :func:`set_event_loop`, and
-:func:`new_event_loop` functions can be customized.
+An event loop policy is a global (per-interpreter) object
```

Comment:
> An event loop policy is a global (per-interpreter) 

I know the details here are hard to understand but "per-interpreter" can give false sense that `asyncio` can work under multiple interpreters. It doesn't since it caches the thread id and does not checks the interp id. It would be better to remove `per-interpreter` altogether here.

See https://github.com/python/cpython/issues/91375

Reply:
Oh, you're right, I remember now. We should add a note to that issue to add "per-interpreter" back here once that's been fixed. (It's not rocket science to fix it, IIRC, just work?)

Reply:
Oh and since this is already merged we'd need a new PR. :-(

Reply:
Okay done https://github.com/python/cpython/pull/97618